### ISSUE TITLE
Not able to install the package

### ISSUE THREAD

**Issue by tuggawaugh** [ropensci/rtweet#345]
### Problem

I am not able to install the package. Gives a compilation fail error. No libraries are available.
I have set and updated working directories but problem persist. I have also restarted R as well as PC multiple times but still i get the same error and am not able to install the package.

Below is the screen shot with session ID. 
Can you help?

### OUTPUT 
ERROR:
install.packages("rtweet")

  There is a binary version available but the source version is later:
       binary source needs_compilation
rtweet  0.6.8  0.6.9             FALSE

installing the source package ‘rtweet’

trying URL 'https://cran.rstudio.com/src/contrib/rtweet_0.6.9.tar.gz'
Content type 'application/x-gzip' length 3410296 bytes (3.3 MB)
downloaded 3.3 MB

* installing *source* package 'rtweet' ...
** package 'rtweet' successfully unpacked and MD5 sums checked
** libs
Warning: running command 'make -f "C:/Users/HARSHI~1.SHA/DOCUME~1/R/R-34~1.2/etc/x64/Makeconf" -f "C:/Users/HARSHI~1.SHA/DOCUME~1/R/R-34~1.2/share/make/winshlib.mk" SHLIB_LDFLAGS='$(SHLIB_CXXLDFLAGS)' SHLIB_LD='$(SHLIB_CXXLD)' SHLIB="rtweet.dll" WIN=64 TCLBIN=64 OBJECTS="RcppExports.o unroll_connections.o"' had status 127
ERROR: compilation failed for package 'rtweet'
* removing 'C:/Users/harshil.b.shah/Documents/R/R-3.4.2/library/rtweet'
Warning in install.packages :
  running command '"C:/Users/HARSHI~1.SHA/DOCUME~1/R/R-34~1.2/bin/x64/R" CMD INSTALL -l "C:\Users\harshil.b.shah\Documents\R\R-3.4.2\library" C:\Users\HARSHI~1.SHA\AppData\Local\Temp\RtmpoNtx13/downloaded_packages/rtweet_0.6.9.tar.gz' had status 1
Warning in install.packages :
  installation of package ‘rtweet’ had non-zero exit status

The downloaded source packages are in
	‘C:\Users\harshil.b.shah\AppData\Local\Temp\RtmpoNtx13\downloaded_packages’
> packageVersion("rtweet")
Error in packageVersion("rtweet") : package ‘rtweet’ not found

### Session info

R version 3.4.2 (2017-09-28)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows >= 8 x64 (build 9200)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252  LC_CTYPE=English_United States.1252   
[3] LC_MONETARY=English_United States.1252 LC_NUMERIC=C                          
[5] LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] twitteR_1.1.9

loaded via a namespace (and not attached):
[1] bit_1.1-12     httr_1.3.1     compiler_3.4.2 rjson_0.2.15   R6_2.2.2       DBI_0.7        tools_3.4.2   
[8] yaml_2.1.15    bit64_0.9-7   

![Rtweet error](https://user-images.githubusercontent.com/43910610/61584865-72cdc500-ab1d-11e9-8d12-c2f6050d4fc7.JPG)

**Comment by ychodagam** [comment]
I am facing the same issue

**Comment by JBGruber** [comment]
# tl;dr

You have to either update your `R` version or install [`RTools`](https://cran.r-project.org/bin/windows/Rtools/) to be able to install `rtweet`. 

# Long version

The Windows operating system lacks some programs you need to compile packages (which needs to be done when installing from source). To get these programs, you need to install [`RTools`](https://cran.r-project.org/bin/windows/Rtools/). 

Usually users don't even notice this though as most packages are pre-compiled on CRAN servers and you can download the binaries directly. Look on the [CRAN page for `rtweet`](https://cran.r-project.org/web/packages/rtweet/), for example:

![image](https://user-images.githubusercontent.com/23524101/62351114-f667b880-b504-11e9-97a7-e630ac46c8b7.png)

Now as you can see, there are three versions of the binaries: r-devel, r-release and r-oldrel. The current release is 3.6.1. That means that r-devel is `R` 3.7.0 (the current development version) and r-oldrel is 3.5.3 (the last release with the minor 5). Since you are using version 3.4.2 (which is almost 2 years old), there is no binary for your `R` version, which means `R` automatically tries to compile the package. Solution as described above.

**Comment by tuggawaugh** [comment]
This worked, thank you very much.
